processor/processor.go

```diff
@@ -2360,6 +2372,13 @@ func (proc *Handle) Store(partition string, in *storeMessage) {
 				}
 			}
 
+			if proc.isTrackedUsersCollectionEnabled() {
```

Comment:
```suggestion
```
No need for a check here, we can use Nop implementation to disable MTU tracking without adding this logic inside processor.